pkg/governance-scripts/contracts/20220415-veBAL-L2/veBALL2GaugeSetupCoordinator.sol

```diff
@@ -49,13 +50,15 @@ contract veBALL2GaugeSetupCoordinator is ReentrancyGuard {
     IVotingEscrow private immutable _votingEscrow;
     IGaugeController private immutable _gaugeController;
     IGaugeAdder private immutable _gaugeAdder;
-    ILiquidityGaugeFactory private immutable _ethereumGaugeFactory;
+    IBaseRootGaugeFactory private immutable _ethereumGaugeFactory;
```

Comment:
With https://github.com/balancer-labs/balancer-v2-monorepo/pull/1633 merged, we'll no longer need to worry about this.